taipy/gui_core/_adapters.py

```diff
@@ -289,7 +289,7 @@ def get_hash():
     def get_type(attr: str):
         if prop := _get_datanode_property(attr):
             return _GuiCoreScenarioProperties.__DN_TYPES.get(prop, "any")
-        return _GuiCoreScenarioProperties._SC_TYPES.get(attr, "any")
+        return _GuiCoreScenarioProperties._SC_TYPES.get(attr, {"type": "any"}).get("type", "any")
```

Comment:
That reads really strange: if 'attr' is unknown, you carry on anyway?
With my renaming I would propose `_SC_PROPS.get(attr).get("type", "any")`

Reply:
Support for custom properties!

Reply:
Exxxxxxxxxxcellent point